Extension/src/LanguageServer/extension.ts

```diff
@@ -541,6 +543,63 @@ async function selectIntelliSenseConfiguration(sender?: any): Promise<void> {
     return clients.ActiveClient.promptSelectIntelliSenseConfiguration(true, sender);
 }
 
+async function installCompiler(sender?: any): Promise<void> {
+    const telemetryProperties = { sender: util.getSenderType(sender), platform: os.platform(), ranCommand: 'false' };
+    const ok = localize('ok', 'OK');
+    switch (os.platform()) {
+        case "win32":
+            showInstallCompilerWalkthrough();
+            break;
+        case "darwin": {
+            const title = localize('install.compiler.mac.title', 'The clang compiler will now be installed');
+            const detail = localize('install.compiler.mac.detail', 'You may need to type your password in the terminal window that will appear to authorize the installation.');
```

Comment:
Should this be re-worded to say you may be prompted to type your password? Simple saying "You may need to type your password" seems like it could lead to some users typing their password into the terminal when they are not prompted and should not?

Reply:
@bobbrow Did you see this comment?

Reply:
I changed the text to say they may be prompted. I don't think I can know 100% what the prompt will look like, but it will be in the VS Code terminal.  Do you want me to say that?